isotptun.c

```diff
@@ -154,9 +177,9 @@ int main(int argc, char **argv)
 			else if (elements == 2)
 				opts.flags |= (CAN_ISOTP_EXTEND_ADDR | CAN_ISOTP_RX_EXT_ADDR);
 			else {
-				printf("incorrect extended addr values '%s'.\n", optarg);
+				fprintf(stderr, "incorrect extended addr values '%s'.\n", optarg);
 				print_usage(basename(argv[0]));
-				exit(0);
+				exit(EXIT_FAILURE);
 			}
```

Comment:
I'm a tad bit uncertain I get this right. Single patch separate from the daemon feature? Can do like that as well. Clear communication is so difficult... :-)

Then there are the changes to print out errors to stderr instead of stdout. Opinion on them?